storage/series_test.go

```diff
@@ -177,3 +178,68 @@ func TestSeriesFlushToDisk(t *testing.T) {
 	err := series.FlushToDisk(writer, flushTime, segmentHolder)
 	require.Nil(t, err)
 }
+
+func TestSeriesTickEmptySeries(t *testing.T) {
+	opts := seriesTestOptions()
+	series := newDatabaseSeries("foo", bootstrapped, opts).(*dbSeries)
+	err := series.Tick()
+	require.Equal(t, errSeriesAllDatapointsExpired, err)
+}
+
+func TestSeriesTickNeedsDrain(t *testing.T) {
+	ctrl := gomock.NewController(t)
+	defer ctrl.Finish()
+
+	opts := seriesTestOptions()
+	curr := time.Now()
+	start := curr
+	opts = opts.NowFn(func() time.Time {
+		return curr
+	})
+	series := newDatabaseSeries("foo", bootstrapped, opts).(*dbSeries)
+	series.blocks.AddBlock(NewDatabaseBlock(start, nil, opts))
```

Comment:
Is there a reason this test needs a database block? (curious)  If it's because otherwise it would be empty could we not just write to the buffer a single value?